Lib9c/Battle/Simulator.cs

```diff
@@ -62,6 +62,47 @@ MaterialItemSheet materialItemSheet
         {
             var reward = new List<ItemBase>();
 
+            while (reward.Count < maxCount)
+            {
+                try
+                {
+                    var data = itemSelector.SelectV2(1).First();
```

Comment:
Isn't it correct to use `itemSelector.SelectV2()` in `SetRewardV2()`?

Reply:
nope, `SetRewardV2` to use in `SelectV3` is correct. because `SetReward` is being used for older actions.(like `HackAndSlash, HackAndSlash2, HackAndSlash3`)

Reply:
Oh I was wondering why there is no more logic to use `itemSelector.Select()` in `SetReward()` method.
Looking at the log, it was confirmed that `itemSelector.Select()` was uploaded to `itemSelector.SelectV2()` without upgrading `SetReward()` in the f58f998 commit.